src/components/LocationMarker/index.js

```diff
@@ -38,19 +38,30 @@ const LocationMarker = ({ geolocation }) => {
 
   const { accuracy, latitude, longitude } = geolocation
 
+  const removeLocationDot = () => {
+    locationDot.remove()
+  }
+
   useEffect(() => {
     if (!mapInstance) return undefined
 
+    const addLocationDot = () => {
+      locationDot.setLatLng([latitude, longitude])
+      locationDot.addTo(mapInstance)
+    }
+
```

Comment:
Either move this function out of the scope of the `LocationMarker` component or wrap it in a `useCallback`.

Reply:
✅